docs/_docs/inversion.md

```diff
@@ -24,7 +24,9 @@ When this is enabled, developers only have to take one more step to have full in
   </pre>
 
   <pre lang="swift" class = "swiftCode hidden">
```

Comment:
I think you can now remove the `hidden` class now as Swift is available.

Reply:
Oh I got it, thankx